src/components/content-tab-import-export.tsx

```diff
@@ -1,6 +1,138 @@
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import { getIpcApi } from '../lib/get-ipc-api';
+import { ExportOptions } from '../lib/import-export/export/types';
+import { BackupArchiveInfo } from '../lib/import-export/import/types';
+
 interface ContentTabImportExportProps {
 	selectedSite: SiteDetails;
 }
-export function ContentTabImportExport( _props: ContentTabImportExportProps ) {
-	return null;
+
+export const ExportSite = ( {
+	selectedSite,
+	onExport,
+}: {
+	selectedSite: SiteDetails;
+	onExport: ( options: ExportOptions ) => Promise< void >;
+} ) => {
+	const onExportFullSite = async () => {
+		const path = await getIpcApi().showSaveAsDialog( {
+			title: __( 'Save backup file' ),
+			defaultPath: `${ selectedSite.name.split( ' ' ).join( '_' ).toLowerCase() }.tar.gz`,
```

Comment:
Jetpack Backup uses names like `jetpack-backup-naruniec-me-2024-07-12-00-51-41.tar.gz`.

What do you think about following a similar format, e.g., `studio-backup-my-prime-website-2024-07-12-00-51-41.tar.gz`?